### PR TITLE
LGTM deprecation: a few more references missed in earlier PRs

### PR DESCRIPTION
**Pull request by felicitymay**
Now that it's easier to search for the remaining references to LGTM, I found a few more in these articles. Apologies to the Java and JavaScript teams for missing these articles in the earlier PRs.

### CODE REVIEW

**Review by atorralba — APPROVED** [review]
Java LGTM 👍

**Review by erik-krogh — APPROVED** [review]

File: docs/codeql/codeql-language-guides/using-flow-labels-for-precise-data-flow-analysis.rst [hunk 1]
```diff
@@ -391,10 +390,10 @@ Some of our standard security queries use flow labels. You can look at their imp
 to get a feeling for how to use flow labels in practice.
 
 In particular, both of the examples mentioned in the section on limitations of basic data flow above
-are from standard security queries that use flow labels. The `Prototype pollution
-<https://lgtm.com/rules/1508857356317>`_ query uses two flow labels to distinguish completely
+are from standard security queries that use flow labels. The `Prototype-pollutiing merge call
```
Comment (stevecat) [inline docs/codeql/codeql-language-guides/using-flow-labels-for-precise-data-flow-analysis.rst:393]:
```suggestion
are from standard security queries that use flow labels. The `Prototype-polluting merge call
```

**Review by stevecat — APPROVED** [review]
Looks good! Just one little typo ✨

**Comment by felicitymay** [comment]
Retargeted to release candidate branch

File: docs/codeql/codeql-language-guides/using-type-tracking-for-api-modeling.rst [hunk 2]
```diff
@@ -228,15 +228,23 @@ Here's see an example of what this can handle now:
 
 Tracking in the whole model
 ---------------------------
-We applied this pattern to ``firebaseDatabase()`` in the previous section, and it
-can just as easily apply to the other predicates.
-For reference, here's our simple Firebase model with type tracking on every predicate:
+We applied this pattern to ``firebaseDatabase()`` in the previous section, and we can
```
[the remark this answers is not in the document]
Comment (asgerf) [reply docs/codeql/codeql-language-guides/using-type-tracking-for-api-modeling.rst:231]: > Note that this article uses https://github.com/googlearchive/friendlypix-web throughout.

Now that the query console links have been removed, it shouldn't reference that repository in any way.

And yes the article is still useful, no revision is needed as a result of that repo being archived.

[the remark this answers is not in the document]
Comment (felicitymay) [reply docs/codeql/codeql-language-guides/using-type-tracking-for-api-modeling.rst:231]: Many thanks for checking @asgerf

**Comment by felicitymay** [comment]
Thanks for the reviews ✨

**Pull request merged**